crates/sui-rpc-api/src/client/mod.rs

```diff
@@ -136,6 +136,7 @@ impl Client {
 
         let (metadata, response, _extentions) = self
             .raw_client()
+            .max_decoding_message_size(64 * 1024 * 1024)
```

Comment:
And given the maximum size of a checkpoint is ~32MB (sans objects), lets set the maximum decode size of the fullcheckpoint api to 64MB to give us some more headroom